tests/test_server/test_auth/test_keycloak_auth.py

```diff
@@ -1,3 +1,4 @@
+import base64
```

Comment:
Please add tests for `/users/me` endpoint. Probably there is a test for some other endpoint, like `/datasets`, handling auth errors, we can update it to use new endpoint